app/compass_svc.py

```diff
@@ -72,3 +76,62 @@ def _get_layer_boilerplate(name, description):
             layer['techniques'].append(technique)
 
         return web.json_response(layer)
+
+    @staticmethod
+    def extract_techniques(request_body):
+        techniques = request_body.get('techniques')
+        adversary_techniques = set()
+        for technique in techniques:
+            if technique.get('score') > 0:
```

Comment:
why do we care about the score here?

Reply:
Score determines what technique is included. Can also include layer techniques that are colored. Score seemed the most straightforward and standard way of using layers and avoid the use of interpreting the meaning colors. (Maybe a red colored technique means DONT INCLUDE)

Reply:
ah interesting.. ok. this probably justifies a comment